Xamarin.Forms.Platform.Android/Cells/BaseCellView.cs

```diff
@@ -190,7 +190,10 @@ public void SetRenderHeight(double height)
 			Bitmap bitmap = null;
 			IImageSourceHandler handler;
 
-			if (source != null && (handler = Registrar.Registered.GetHandler<IImageSourceHandler>(source.GetType())) != null)
+			var rt = source as System.Reflection.IReflectableType;
+			var t = rt != null ? rt.GetTypeInfo().AsType() : source.GetType();
+
+			if (source != null && (handler = Registrar.Registered.GetHandler<IImageSourceHandler>(t)) != null)
```

Comment:
the check for `source != null` happens to late. L194 is potentially accessing a `null` instance

what about 
```
var t = (source as IReflectableType) ?.GetTypeInfo().AsType() ?? source?.GetType();
if (t != null && ....)
```